### PR TITLE
Letting people know GO is a requirement

### PR DESCRIPTION
**Pull request by rcsavage**

### CODE REVIEW

**Comment by devimc** [comment]
thanks for raising this, can you add a body to your commit message and file an issue? more information https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#patch-format

File: kernel/README.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 This document explains the steps to build a compatible kernel with Kata
 Containers. To do this use build-kernel.sh, this script automates the
-process to build a kernel for Kata Containers.
```
Comment (grahamwhaley) [inline kernel/README.md]: Nitpick - but I think you've accidentally added an extra space (` `) to the end of this line - I was wondering why it turned up in the patch ;-)

File: kernel/README.md [hunk 2]
```diff
@@ -2,7 +2,11 @@
 
 This document explains the steps to build a compatible kernel with Kata
 Containers. To do this use build-kernel.sh, this script automates the
-process to build a kernel for Kata Containers. 
+process to build a kernel for Kata Containers.
+
+## Requirements
+
+Go-lang version 1.10+ required by the setup.sh script.
```
Comment (grahamwhaley) [inline kernel/README.md]: Personally I'd phrase it as something like '[Go](https://golang.org/) version 1.10 or above required'. Just my preference though.

Comment (jodh-intel) [reply kernel/README.md]: Hi @rcsavage - thanks for raising this. However, I'd prefer we don't duplicate versions like this. Instead, please could you refer to the following which explains the golang requirement and points the user to the "versions database" file which specifies all versions of external entities used by Kata:

 -https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#requirements-to-build-individual-components

**Comment by jodh-intel** [comment]
Hi @rcsavage - please could you let us know if you plan to update this PR?

**Comment by caoruidong** [comment]
ping @rcsavage

**Comment by sboeuf** [comment]
@rcsavage hi! thanks for the PR, we would like to know if you're planning on updating this, otherwise we'll end up closing it.

**Comment by jodh-intel** [comment]
Ping @rcsavage.

**Comment by grahamwhaley** [comment]
Hi @rcsavage - can you let us know if you will be able to re-visit and complete this pull request you submitted please?
As it stands, it costs our review team some time each week to check if there has been any progress. If you cannot commit to landing this PR, just let us know, and we will take it under our [Assisted PR workflow](https://github.com/kata-containers/community/blob/master/CONTRIBUTING.md#assisted-pr-workflow) process to clean up and land it.
thanks!

**Comment by grahamwhaley** [comment]
@rcsavage - I've re-worked this PR of yours and submitted it as #479 - please feel free to review that.
Once that has landed then we'll close this PR.
thanks!

**Comment by marcov** [comment]
I am closing this PR, as it was superseded by #479 that's already merged.
Thanks @rcsavage !

**Pull request closed without merging**